pkg/controllers/management/clusterprovisioner/driver.go

```diff
@@ -12,18 +12,30 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-const DriverNameField = "driverName"
+const (
+	DriverNameField = "driverName"
+
+	DefaultBackupIntervalHours = 12
+	DefaultBackupRetention     = 6
+)
 
 func (p *Provisioner) driverCreate(cluster *v3.Cluster, spec v3.ClusterSpec) (api string, token string, cert string, err error) {
 	ctx, logger := clusterprovisioninglogger.NewLogger(p.Clusters, cluster, v3.ClusterConditionProvisioned)
 	defer logger.Close()
 
 	spec = cleanRKE(spec)
+	if cluster.Spec.RancherKubernetesEngineConfig != nil &&
+		cluster.Spec.RancherKubernetesEngineConfig.Services.Etcd.BackupConfig == nil { // no backup config
```

Comment:
we should enable it on the api layer, meaning that the field has to be set on the api layer, and persisted to etcd